common/tools/dev-tool/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES2015",
+    "target": "ES2017",
```

Comment:
This tsconfig for dev-tool is `noEmit`, so does it even need a target (or `module` for that matter)?

Reply:
It's `noEmit`, but doesn't `ts-node` still use tsconfig to decide how to compile it so Node can interpret?